server/src/optima/project.py

```diff
@@ -259,8 +260,7 @@ def getProjectList():
         # Get projects for current user
         projects = ProjectDb.query.filter_by(user_id=current_user.id)
         for project in projects:
```

Comment:
it should be just project.data_upload_time() (considering the changes made there)